index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>calculator</title>
+  </head>
+  <body>
+    <div>
+      <input type="text" id="screen" value="" currentOperation="" disabled />
+    </div>
+
+    <button id="clearButton">AC</button>
+    <button id="previousAnswerButton">PreviousAns</button>
+    <button id="getHistory">HIS</button>
+    <button id="operationButtons">/</button>
+
+    <br />
+    <button numbersToPress>1</button>
+    <button numbersToPress>2</button>
+    <button numbersToPress>3</button>
+    <button id="operationButtons">*</button>
```

Comment:
It's not proper HTML to have multiple elements on the screen with the same ID

Reply:
Good point. Changed it to being a class.